index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"/>
+    <script src="script.js" defer></script>
+    <title>Main Page</title>
+</head>
+<body>
+    <header>
+        <div id="navbar-social-icons">
+                <i class="fab fa-facebook-f"></i>
+                <i class="fab fa-twitter"></i>
+                <span>English</span>
+                <span>My Page</span>
+                <span>Logout</span>
+            </div>
+        <nav class="navbar">
+            <div class="hamburger-menu">
+                <div class="hamburger-items1"></div>
+                <div class="hamburger-items2"></div>
+                <div class="hamburger-items3"></div>
+            </div>
+            <img id="logo" src="img/myLogo.png" alt="">
+            <ul class="menu-list">
+                <li><a class="menu-list-items" href="#">Home</a></li>
+                <li><a class="menu-list-items" href="#program">Program</a></li>
+                <li><a class="menu-list-items" href="about.html">About</a></li>
+                <li><a class="menu-list-items" href="#">Sponsors</a></li>
+                <li><a class="menu-list-items" href="#">News</a></li>
+                <li><a class="menu-list-items campaign" href="#">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="introduction">
+            <h1 class="intro-text">A New Wave of English Language exams on your Campus - IELTS</h1>
+            <h2 class="intro-text2">2011 NCTA Annual Conference San Diego, CA</h2>
+            <a class="ielts-link" rel="noopener noreferrer" href="https://www.ielts.org/" target="_blank">www.ielts.org</a>
+        </section>
+        <section class="our-program" id="program">
+            <h3 class="our-program-header">Our program</h3>
+            <div class="our-program-cards-container">
```

Comment:
- [ ] OPTIONAL: Please consider giving the fixed space for all cells in Our program section:

Currently:  | Aligned with grid
------------- | -------------
<img width="342" alt="Screenshot 2022-03-09 at 23 54 18" src="https://user-images.githubusercontent.com/77038610/157523129-cdf3cd3a-75b5-41dd-ae95-bf52e70c0cf3.png">  | <img width="409" alt="Screenshot 2022-03-10 at 00 01 37" src="https://user-images.githubusercontent.com/77038610/157524258-f5f5f49d-e705-46af-b70a-5fd73aa84382.png">


You can easily do it by setting:  
```
.our-program-cards {
    display: grid;
    grid-template-columns: 1fr 2fr 5fr;
}
```

Reply:
@ShoiraTa Thanks for giving hint to fix this

Reply:
This is an optional requirement,  you have done a great job. You only need to update the readme!

Reply:
@ShoiraTa I will try this because it improves the project